index.html

```diff
@@ -56,83 +63,139 @@ <h2>LET'S CONNECT</h2>
-            <div class="lang-title">
-              <h2>Languages</h2>
-              <ul>
-              <li class="arrow-down"><i class="fa fa-arrow-down"></i></li>
-              </ul>
-            </div>
-            <div class="image-language">
-              <img src="images/web.png" alt="web images">
-            </div>
-            <div class="skills-language">
-              <h3>Framework</h3>
-              <ul>
-                <li class="arrow-right"><i class="fa fa-arrow-right"></i></li>
-                </ul>
-            </div>
-              <hr>
-              <div class="skills-info">
-              <h4 class="skills-title">skills</h4>
-              <ul>  
-              <li class="arrow-side"><i class="fa fa-arrow-right"></i></li>
-              </ul>
-      </div>
-              <hr>
-              </div>
-            </div>
+    <!--first card-->
+              <section class="portfolio">
+                <article class="card">
+                  <div class="card-container">
+                    <img src="images/snapshoot-portfolio.svg" alt="">
+                    <div class="card-content">
+                    <h3>tonic</h3>
+                    <h4><span>conopy</span>&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      back end dev&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      2015
+                    </h4>
+                    <p>a daily selection of privately personalized reads; no accounts or
+                      sign-ups required.</p>
+                    <ul>
+                      <li>html</li>
+                      <li>Ruby on Rails</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button>see project</button>
+                    </div>
+                  </div>
+                </article>
+                <article class="card">
+                  <div class="card-container-1">
+                    <img src="images/snapshoot-portfolio-1.svg" alt="">
+                    <div class="card-content-1">
+                    <h3>Facebook 360</h3>
+                    <h4><span>facebook</span>&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      full stack Dev&nbsp;
+                      <img src="images/counter.svg" alt="">&nbsp;
+                      2015
+                    </h4>
+                    <p>Exploring the future of media in facebook's first virtual Reality <br>
+                      app a place to discover and enjoy 360 points and videos on <br>
+                      Gear VR.</p>
+                    <ul>
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javascript</li>
+                    </ul>
+                    <button>see project</button>
+                    </div>
+                  </div>
+                </article>
```

Comment:
your card doesn't have the same padding as the one in the Figma design please try to change your card to be the same as the one in the Figma design as the previous reviewer asked

- [ ] Your card
![image](https://user-images.githubusercontent.com/61790587/165809205-5cff67da-9c8a-465c-b332-4dda920b152c.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/165809383-621a6d67-efc6-41da-ac2e-f66061400f8e.png)